translations/es.json

```diff
@@ -0,0 +1,742 @@
+			"message": "Conecta tu wallet para empezar a hacer trading"
+		},
+		"footer-card": {
+			"market-closure": {
+				"button-label": "mercado cerrado",
+				"reasons": {
+					"frozen": {
+						"message": "El synth {{currencyKey}} ha sido congelado - reajuste en progreso."
+					},
+					"market-closure": {
+						"message": "El mercado {{currencyKey}} ha sido cerrado - reajuste en progreso."
+					},
+					"system-upgrade": {
+						"message": "Synth {{currencyKey}} pausado. Por favor consulte el Discord de Synthetix para más detalles."
+					},
+					"circuit-breaker": {
+						"message": "Cortacircuito activado para {{currencyKey}} - reajuste en progreso."
+					},
+					"emergency": {
+						"message": "Apagado de emergencia activado para {{currencyKey}} - reajuste en progreso."
+					},
+					"both-synths-suspended": {
+						"message": "Los Synths seleccionados están actualmente pausados - reajuste en progreso."
+					},
+					"after-hours": {
+						"message": "El mercado de valores está cerrado. Haz trading durante las horas de cierre en Balancer",
+						"button-label": "Abrir la pool de Balancer"
+					}
+				}
+			},
+			"get-l2-gas": {
+				"message": "No hay suficiente wETH de L2 para las tarifas (fees) de la transacción. Adquiere un poco vía Optimism Bridge:"
+			}
+		},
+		"common": {
+			"from": "de",
+			"into": "a"
+		},
+		"errors": {
+			"fee-reclamation": "Hay un periodo de prueba tras completar un trade. Por favor, espere aproximadamente {{waitingPeriod}} minutos antes de intentar intercambiar {{currencyKey}}"
+		},
+		"charts": {
+			"single": "simple",
+			"double": "doble"
+		}
+	},
+	"dashboard": {
+		"page-title": "Dashboard | Kwenta",
+		"trending": "Tendencia",
+		"watchlist": {
+			"title": "Watchlist"
+		},
+		"tabs": {
+			"nav": {
+				"synth-balances": "Balances de Synths",
+				"convert": "Conversor de Synths",
+				"crypto-balances": "Balances de criptomonedas",
+				"transactions": "Transacciones",
+				"deprecated": "Deprecado"
+			}
+		},
+		"onboard": {
+			"convert": "Synth Onramp",
+			"title": "Aprende como funciona",
+			"subtitle": "Empieza con Kwenta y haz trades sobre cualquier activo.",
+			"learn-more": "Leer más",
+			"how-it-works-modal": {
+				"title": "Como funciona"
+			},
+			"convert-card": {
+				"slippage-selector": {
+					"title": "Slippage"
+				}
+			}
+		},
+		"your-portfolio": {
+			"title": "Tu Portfolio"
+		},
+		"synth-sort": {
+			"price": "Precio",
+			"24h-vol": "Volumen 24H ",
+			"24h-high": "Máximo 24H",
+			"24h-low": "Mínimo 24H",
+			"24h-change": "Cambio 24H"
+		},
+		"transactions": {
+			"synth-sort": {
+				"allSynths": "Todos los Synths"
+			},
+			"order-type-sort": {
+				"allOrderTypes": "Todos los tipos de órdenes",
+				"market": "Mercado",
+				"limit": "Límite"
+			},
+			"table": {
+				"orderType": "Tipo de órden",
+				"from": "De",
+				"to": "A",
+				"value": "Valor",
+				"fee-reclaim": "Reembolso/Reclamación de la comisión (fee)",
```

Comment:
Possibility to shorten this so we avoid a line break (see screenshot)? Can't think of a shorter term though. 

I would suggest to simply remove Reclamación, it still should be clear anyway what the item is for if it says ``Reembolso de la comisión (fee)``.

![Screenshot_2021-10-13 Kwenta](https://user-images.githubusercontent.com/548702/137190047-8acbef0d-e738-472f-a88a-e20acab64fd7.png)